test/evergreen.yml

```diff
@@ -3393,6 +3393,33 @@ tasks:
           perf-test-name: many-table-stress-backup
           maxruns: 1
 
+  - name: perf-test-evict-fairness
```

Comment:
In the Jenkins jobs this test would fail when we didn't have exactly 200 evicted tables
```
if [ $count -eq 200 ]; then
    echo "eviction walk fairness pass"
else
    echo "eviction walk fairness fail"
    exit 1
fi
```
but I don't see this check being performed anymore. I don't think we want to add assertions like this into the perf testing framework so let's discuss with @jeremythorp how we implement this

Reply:
Yup, I agree, a discussion on this is required. My suggestion is to simply add a function in `evergreen.yml`

Reply:
@ajmorton , well spotted! Please can you create a ticket to add this test (I suggest we do it outside this PR).  Let's get this PR merged (once it's ready, of course :-) ) and then address this. 
As @Siddhartha8899 suggests, we can discuss the best way to do it.

Reply:
@ajmorton , well spotted! Please can you create a ticket to add this test (I suggest we do it outside this PR). Let's get this PR merged (once it's ready, of course :-) ) and then address this.
As @Siddhartha8899 suggests, we can discuss the best way to do it.

Reply:
👍 the new ticket [WT-8363](https://jira.mongodb.org/browse/WT-8363) has been created to track this